tests/unit/test_row_noise.py

```diff
@@ -61,10 +61,9 @@ def test_do_not_respond(
     mocker.patch(
         "pseudopeople.noise_level._get_census_omission_noise_levels",
         side_effect=(
-            lambda *_: config.get_value(
+            lambda *_: config.get_row_probability(
```

Comment:
Nit: for this and the next two cases, if you remove the trailing comma, black will allow the arguments to stay on one line.